cmd/queue/main.go

```diff
@@ -255,8 +258,16 @@ func probeQueueHealthPath(port int, timeoutSeconds int) error {
 	// Using PollImmediateUntil instead of PollImmediate because if timeout is reached while waiting for first
 	// invocation of conditionFunc, it exits immediately without trying for a second time.
 	timeoutErr := wait.PollImmediateUntil(aggressivePollInterval, func() (bool, error) {
-		var res *http.Response
-		if res, lastErr = httpClient.Get(url); res == nil {
+		req, lastErr := http.NewRequest(http.MethodGet, url, nil)
```

Comment:
I think you're creating a new lastErr variable here in the inner scope. The outer one remains nil. Same below.

Reply:
Good caught. Done.